tests/Doctrine/Tests/ORM/QueryTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Doctrine\Tests\ORM;
+
+use Doctrine\Tests\Models\CMS\CmsUser;
+use Doctrine\Tests\OrmTestCase;
+
+/**
+ * Test case for the Query class
+ *
+ * @author Konstantin Kuklin <konstantin.kuklin@gmail.com>
+ */
+class QueryTest extends OrmTestCase
+{
+    /**
+     * @var \Doctrine\ORM\EntityManager
+     */
+    private $_em;
+
+    protected function setUp()
+    {
+        $this->_em = $this->_getTestEntityManager();
+    }
+
+    /**
+     * setResultCacheProfile should works correct with null
+     */
+    public function testSetResultCacheProfile()
```

Comment:
Shouldn't this be part of https://github.com/doctrine/doctrine2/blob/f2dc9a8f92b7eb9a566b3ba7af16559cc2bc0eb7/tests/Doctrine/Tests/ORM/Query/QueryTest.php?